features/fork.feature

```diff
@@ -202,3 +202,17 @@ Scenario: Related fork already exists
     And "git.my.org" is a whitelisted Enterprise host
     When I successfully run `hub fork`
     Then the url for "mislav" should be "git@git.my.org:mislav/dotfiles.git"
+
+  Scenario: Fork a repo to a specific organization
+    Given the GitHub API server:
+      """
+      get('/repos/acme/dotfiles') { 404 }
+      post('/repos/evilchelu/dotfiles/forks') {
+        assert :organization => "acme"
```

Comment:
This is good. It just occurred to me that we should add an assert to one of the vanilla scenarios (no `--org` parameter) that `:organization` is nil. Could you add that too?